code/game/gamemodes/nations/nations.dm

```diff
@@ -0,0 +1,164 @@
+datum/game_mode/nations
+	name = "nations"
+	config_tag = "nations"
+	required_players_secret = 25
+	var/const/waittime_l = 1200 //lower bound on time before intercept arrives (in tenths of seconds)
+	var/const/waittime_h = 3000 //upper bound on time before intercept arrives (in tenths of seconds)
+	var/kickoff = 0
+	var/victory = 0
+	var/list/cargonians = list("Quartermaster","Cargo Technician","Shaft Miner")
+
+
+/datum/game_mode/nations/post_setup()
+	spawn (rand(waittime_l, waittime_h))
+		spawn(50)
+			kickoff=1
+			send_intercept()
+			split_teams()
+			for(var/mob/M in player_list)
+				if(!istype(M,/mob/new_player))
+					M << sound('sound/effects/purge_siren.ogg')
+	return ..()
+
+/datum/game_mode/nations/send_intercept()
+	command_announcement.Announce("Due to recent and COMPLETELY UNFOUNDED allegations of massive fraud and insider trading \
+					affecting trillions of investors, the Nanotrasen Corporation has decided to liquidate all \
+					assets of the Centcom Division in order to pay the massive legal fees that will be incurred \
+					during the following centuries long court process. Therefore, all current employment contracts \
+					are IMMEDIATELY TERMINATED. Nanotrasen will be unable to send a rescue shuttle to carry you home,\
+					however they remain willing for the time being to continue trading cargo. Have a pleasant \
+					day.", "FINAL TRANSMISSION, CENTCOM.", new_sound = 'sound/AI/intercept.ogg')
+
+
+/datum/game_mode/nations/proc/split_teams()
+
+	for(var/mob/living/carbon/human/H in player_list)
+		if(H.mind)
+			if(H.mind.assigned_role in engineering_positions)
+				H.mind.nation = all_nations["Atmosia"]
+				H.hud_updateflag |= 1 << NATIONS_HUD
+				var/I = image('icons/mob/hud.dmi', loc = H.mind.current, icon_state = "hudatmosia")
+				H.client.images += I
+				H << "You are now part of the great sovereign nation of [H.mind.nation.name]!"
+				continue
```

Comment:
These should probably have some spacing between them, and having 'else' defined is pointless since they all `continue`